### DISCUSSION TITLE
Is there a way how to navigate among areas using keyboard ? (I mean tree, main editor area and terminal area)

### DISCUSSION THREAD

**Discussion by ratermir** [eclipse-theia/theia#9000] [General]
My idea is to be able to navigate ONLY by kayboard among current (active) components in these areas:
 - object tree area (usually the left side)
 - main editor area (the top center are - the largest one)
 - terminal area (the bottom one)

In every area, current component (the one which is actually visible) should be activated - i.e. active component in the area should nod be changed. 

I didn't find anything similar to this behaviour.

**Comment by kittaakos** [comment]
As far as I know, we do not yet support such accessibility features.
